src/riscv/Cargo.toml

```diff
@@ -64,6 +64,10 @@ features = ["utils"]
 git = "https://gitlab.com/tezos/tezos.git"
 default-features = false
 
+[workspace.dependencies.evm_kernel]
+git = "https://gitlab.com/tezos/tezos.git"
+features = ["debug"]
```

Comment:
Does `debug` need to be on by default? Is this something that should perhaps be toggleable by the kernel crate instead?

Reply:
Indeed, and this dependency shouldn't be here to begin with since kernels have their own workspaces. Moved [here](https://github.com/tezos/riscv-pvm/compare/cfc322b76290aa694761848edd5ef2913fa0dee7..92432d44a244d6b1ec5dd7d566d1d3ba11f8816e).